ga4mp/store.py

```diff
@@ -0,0 +1,125 @@
+import json
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+class BaseStore(dict):
+    def __init__(self):
+        self = {
```

Comment:
Confirmed that this works as expected using a stripped down test:
```py
class Test(dict):
    def __init__(self):
        self.update([("user_properties", {}), ("session_parameters", {})])

test = Test()

print(test)
```

Since that worked, I've changed the code accordingly.